.woodpecker/binaries.yml

```diff
@@ -7,9 +7,9 @@ when:
   event: tag
 
 variables:
-  - &golang_image 'golang:1.21.3'
-  - &node_image 'node:21-alpine'
-  - &xgo_image 'techknowlogick/xgo:go-1.21.3'
+  - &golang_image 'docker.io/golang:1.21.3'
```

Comment:
What's the reason to add `docker.io`?

Reply:
The container engine that is executing the pipeline could have other default registries, make it explicit what registry to use for images. See https://www.redhat.com/en/blog/be-careful-when-pulling-images-short-name